styles.css

```diff
@@ -0,0 +1,417 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f1f1f1;
+  margin: 0;
+}
+
+header {
+  background-image: url('image.png');
+  border-bottom-left-radius: 50px;
+  padding: 45px;
+}
+
+nav ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  background-color: white;
+  overflow: hidden;
+}
+
+nav li {
+  float: left;
+}
+
+nav li a {
+  text-decoration: none;
+  color: blue;
+  display: inline-block;
```

Comment:
It is a requirement that your `<nav>` layout should be done with flexbox, kindly fix this 🤗